jbrowse2/jbrowse2.xml

```diff
@@ -31,17 +31,20 @@ cp '$output.files_path/index.html' '$output'
     <metadata>
         <genomes>
             #if str($reference_genome.genome_type_select) == "indexed":
-              <genome path="${reference_genome.genomes.fields.path}" label="${reference_genome.genomes.fields.name}">
-                  <metadata />
+              <genome path="${reference_genome.genome.fields.path}" label="${reference_genome.genome.fields.name}">
+                  <metadata>
+                     <dataset
+                      dname = "${reference_genome.genome.name}" />
+                  </metadata>
               </genome>
             #else
-              <genome path="$reference_genome.genome" label="${reference_genome.genome.element_identifier}">
```

Comment:
element_identifier is always better to use I think.

Reply:
NOT when it's a built in genome - I had never tested one until today and it does not exist.

Reply:
And I forgot to count bigwig so 10 track types working now.
Got any uses for any of those track types? This could do with some testing.
![image](https://github.com/usegalaxy-eu/temporary-tools/assets/6016266/e9f5ba1f-9611-4fb1-9617-88dfbaced458)